app-admin/logstash-bin/files/gentoo-logstash-plugin

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+PLUGINS="/var/lib/logstash/plugins-list.txt"
+LS_INSTALL_DIR="/opt/logstash"
+
+function usage() {
+  echo "Usage: $0 reinstall/install/remove/list plugin..."
+  exit 1
+}
+
+function plugin_install() {
+  while [[ -n "$1" ]]; do
+    echo "$1" >> ${PLUGINS}
+    sort -u "${PLUGINS}" > "${PLUGINS}.tmp"
```

Comment:
I kinda don't get why you sort after adding every plugin instead of adding all of them, and then sorting the file once?